sycl/include/sycl/ext/oneapi/properties/property.hpp

```diff
@@ -63,6 +77,14 @@ template <> struct PropertyToKind<bar_key> {
 // (6.)
 template <> struct IsCompileTimeProperty<bar_key> : std::true_type {};
 
+// (7.)
+template <> struct PropertyMetaName<bar_key::value_t> {
+  static constexpr const char *value = "sycl-bar";
+};
+template <> struct PropertyMetaValue<bar_key::value_t> {
+  static constexpr int value = 5;
+};
```

Comment:
What are these magic values (sycl-bar, 5)?

Reply:
They are just example values to show how to do it, following the guide further up.